lib/templates/markdown.ejs

```diff
@@ -9,8 +9,10 @@
 
 <%- include('pullRequests'); -%>
 
+<% if (commitSummary && commitSummary.length > 0) { -%>
```

Comment:
Is it possible that `commitSummary` is falsy?

Reply:
No it seams not, we can check the length directly